### DISCUSSION TITLE
CommentController.java#L24-L29

### DISCUSSION THREAD

**Discussion by zooneon** [dnd-side-project/dnd-5th-3-backend#137] [코드 리뷰]
https://github.com/dnd-side-project/dnd-5th-3-backend/blob/cb1f6c3d27c0978401e01b9eaf19f9a3634c2933/src/main/java/com/dnd5th3/dnd5th3backend/controller/CommentController.java#L24-L29
ResponseEntity<>를 사용하면 어떤 점이 좋은가요? 혹시 따로 사용하는 이유가 있으신가요?

**Comment by KangSeongHyo** [comment]
https://tecoble.techcourse.co.kr/post/2021-05-10-response-entity/

요기 참고해 보시면 좋을 것 같습니다.

**Reply by zooneon** [reply]
아하 저는 애노테이션만 이용해서 응답 메세지를 처리했는데 ResponseEntity를 이용하면 더 쉽고 깔끔하게 처리할 수 있겠네요!
저도 한 번 수정해보겠습니다 ㅎㅎ
